python/pyspark/ml/tests.py

```diff
@@ -929,6 +932,50 @@ def test_apply_binary_term_freqs(self):
                                    ": expected " + str(expected[i]) + ", got " + str(features[i]))
 
 
+class ALSTest(PySparkTestCase):
+
+    def test_storage_levels(self):
```

Comment:
Yeah both tests are redundant. I guess I was erring on the side of over-caution :) I didn't really want to put a simple test in the doc string, since it's expertParam and probably should stay a bit "hidden".

I can remove these tests, or any other suggestions?